EIPS/eip-7002.md

```diff
@@ -536,10 +527,10 @@ block.header.exits_root = compute_trie_root_from_indexed_data(block.body.exits)
 
 Sketch of spec:
 
-* New operation `ExecutionLayerExit`
-* Will show up in `ExecutionPayload` as an SSZ List bound by length `MAX_EXITS_PER_BLOCK`
-* New function in `process_execution_layer_exit` that has similar functionality to `process_voluntary_exit` but that can fail validations (e.g. validator is already exited) without the block failing (similar to deposit coming from EL)
-* `process_execution_layer_exit` called in `process_operations` for each `ExecutionLayerExit` found in the `ExecutionPayload`
+* New operation `ExecutionLayerWithdrawalRequest`
+* Will show up in `ExecutionPayload` as an SSZ List bound by length `MAX_WITHDRAWAL_REQUESTS_PER_BLOCK`
+* New function in `process_execution_layer_withdrawal_request` that has similar functionality to `process_voluntary_withdrawal_request` but that can fail validations (e.g. validator is already exited) without the block failing (similar to deposit coming from EL)
```

Comment:
There is no `process_voluntary_withdrawal_request` and i think that we can say that a new function is similar to `process_voluntary_exits` because the functionality is extended with partial withdrawals. Maybe just say that the new function can fail validations without block failing and that would be it?